pipelines/docker/kabanero-utils/scripts/enforce_stack_policy.sh

```diff
@@ -85,19 +85,17 @@
         WARNING="[WARNING]"
         ERROR="[ERROR]"
         
-        #Setting insecure image registries
-        #executing the insecure_registry_setup.sh script if exists, to add user mentioned registry url to insecure registry list
-        if [ -f "/workspace/$gitsource/insecure_registry_setup.sh" ]; then
-           echo "$INFO Running the script /workspace/$gitsource/insecure_registry_setup.sh ...."
-           /workspace/$gitsource/insecure_registry_setup.sh
+        # Configure image registry access in the container by adding it to the insecure registry list or enabling TLS verification
+        # by adding it to the trust store based on OpenShift cluster resource configuration.
+        echo "$INFO Running the script /scripts/application_image_registry_access_setup.sh ...."
+        /scripts/application_image_registry_access_setup.sh
+        retVal=$?
+        if [ $retVal -ne 0 ]
+        then
+           echo "$INFO The script failed(/scripts/application_image_registry_access_setup.sh), and the image registry access setup was not complete, aborting the pipelinerun." >&2
```

Comment:
same here.  Should be error here instead of info?

Reply:
updated